flow-model-generator/modules/flow-model-generator-core/src/main/java/io/ballerina/flowmodelgenerator/core/DataMapManager.java

```diff
@@ -2156,14 +2156,12 @@ private String genFunctionDef(WorkspaceManager workspaceManager, Path filePath,
         List<String> paramNames = new ArrayList<>();
         for (Parameter parameter : parameters) {
             String paramName = parameter.type();
```

Comment:
The variable name must be paramTypeName right?

Reply:
Fixed